sumcheck/README.md

```diff
@@ -0,0 +1,16 @@
+# Winter sum-check
+This crate contains an implementation of the sum-check protocol intended to be used for LogUp-GKR by the Winterfell STARK prover and verifier. 
```

Comment:
Could we add a bit more detail here? For example, I'd include the link to the LogUp-GKR paper.

Reply:
Added